python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -384,9 +384,10 @@ def __init__(self, path: str, metadata: Dict = None, num_threads=None):
         self._tensor = torch.from_numpy(ondisk_data)
 
         self._metadata = metadata
-        self._ondisk_npy_array = torch.ops.graphbolt.ondisk_npy_array(
-            path, self._tensor.dtype, self._tensor.shape, num_threads
-        )
+        if torch.ops.graphbolt.detect_io_uring():
+            self._ondisk_npy_array = torch.ops.graphbolt.ondisk_npy_array(
+                path, self._tensor.dtype, self._tensor.shape, num_threads
+            )
```

Comment:
Is this init function correct? Do you still need mmap_mode?

Reply:
mmap mode is served by DiskBasedFeature as well.

Reply:
Right now, there is no capability to read tensor shape, so we need to load with mmap anyway.

Reply:
Without mmap mode, we would read into memory, which we can't do